queue_services/entity-filer/src/entity_filer/worker.py

```diff
@@ -180,6 +180,12 @@ async def process_filing(filing_msg: Dict, flask_app: Flask):  # pylint: disable
 
                 elif filing.get('dissolution'):
                     dissolution.process(business, filing, filing_submission, filing_meta)
+                    AccountService.update_entity(
+                        business_registration=business.identifier,
+                        business_name=business.legal_name,
+                        corp_type_code=business.legal_type,
+                        state=Business.State.HISTORICAL
+                    )
```

Comment:
Do this after `db.session.commit()` below. Refer `changeOfRegistration` if condition on line number 257